vm/src/builtins/super.rs

```diff
@@ -114,6 +114,16 @@ impl PySuper {
         self.typ.clone()
     }
 
+    #[pyproperty(magic)]
+    fn selfclass(&self) -> Option<PyTypeRef> {
+        Some(self.obj.as_ref()?.1.clone())
+    }
+
+    #[pyproperty(name = "__self__")]
+    fn zelf(&self) -> Option<PyObjectRef> {
```

Comment:
```suggestion
    #[pyproperty]
    fn __self__(&self) -> Option<PyObjectRef> {
```
For this case, naming it `__self__` looks better because `zelf` is not a thing in python (though we use it for argument names)